### PR TITLE
Add Rancher ARM64 support docs

### PR DESCRIPTION
**Pull request by jianghang8421**
Related Issue
https://github.com/rancher/docs/issues/1022

### CODE REVIEW

File: content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+The rancher agent image is published using the [docker manifest](https://docs.docker.com/engine/reference/commandline/manifest/), so when you are importing clusters you don't need to change any configurations.
+
+About how to import Kubernetes clusters, please refer to the [Importing Kubernetes Clusters]({{< baseurl >}}/rancher/v2.x/en/cluster-provisioning/imported-clusters/) section.
+
+### Create ARM64 Kubernetes clusters with custom nodes
+
+Rancher supports creating ARM64 platform Kubernetes clusters with your custom nodes. 
+All the images which Rancher using to create Kubernetes clusters are published using the [docker manifest](https://docs.docker.com/engine/reference/commandline/manifest/), so you don't need to change any configurations to create ARM64 platform Kubernetes clusters.
+
+You can check what images rancher is using to create Kubernetes clusters in [this file](https://github.com/rancher/types/blob/master/apis/management.cattle.io/v3/k8s_defaults.go).
```
Comment (loganhz) [inline content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md]: Please remove this. You don't have to show users the code

File: content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+> Currently Rancher only supports using `Flannel` network plugin to create ARM64 platform Kubernetes clusters. 
+> Currently only `v1.12.6-rancher1-2` and `v1.13.4-rancher1-2` versions ARM64 Kubernetes are supported.
+
```
Comment (loganhz) [inline content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md]: Please remove `rancher1-2`. In the future, we will release `rancher1-3` and `rancher1-4`.

File: content/rancher/v2.x/en/arm64-platform/_index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## Running Rancher Server on ARM64 platform
+
+The Rancher image is published using the [docker manifest](https://docs.docker.com/engine/reference/commandline/manifest/), so whether on the AMD64 platform or the ARM64 platform, you can use the same image name and tag to install Rancher.  
+
+About how to install Rancher, please refer to the [Installation]({{< baseurl >}}/rancher/v2.x/en/installation/) section.
+
+## Using Rancher to manage ARM64 Platform Kubernetes
+
+Rancher currently supports the following two ways to manage ARM64 platform kubernetes clusters.
+
```
Comment (loganhz) [inline content/rancher/v2.x/en/arm64-platform/_index.md]: Change `kubernetes` => `Kubernetes`

File: content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+title: Using Rancher to manage ARM64 Kubernetes
+weight: 50
+---
+
+Rancher currently supports the following two ways to manage ARM64 platform kubernetes clusters.
+
```
Comment (loganhz) [inline content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md]: Change `kubernetes` => `Kubernetes`

File: content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md (hunk 1 above)
Comment (jianghang8421) [reply content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md]: Removed

File: content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md (hunk 2 above)
Comment (jianghang8421) [reply content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md]: Removed

File: content/rancher/v2.x/en/arm64-platform/_index.md (hunk 3 above)
Comment (jianghang8421) [reply content/rancher/v2.x/en/arm64-platform/_index.md]: Changed

File: content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md (hunk 4 above)
Comment (jianghang8421) [reply content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md]: Changed

File: content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+About how to import Kubernetes clusters, please refer to the [Importing Kubernetes Clusters]({{< baseurl >}}/rancher/v2.x/en/cluster-provisioning/imported-clusters/) section.
+
+### Create ARM64 Kubernetes clusters with custom nodes
+
+Rancher supports creating ARM64 platform Kubernetes clusters with your custom nodes. 
+All the images which Rancher using to create Kubernetes clusters are published using the [docker manifest](https://docs.docker.com/engine/reference/commandline/manifest/), so you don't need to change any configurations to create ARM64 platform Kubernetes clusters.
+
+> Currently Rancher only supports using `Flannel` network plugin to create ARM64 platform Kubernetes clusters. 
+> Currently only `v1.12.6` and `v1.13.4` versions ARM64 Kubernetes are supported.
+
```
Comment (loganhz) [inline content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md]: > Currently Rancher only supports using `Flannel` network plugin to create Kubernetes clusters on ARM64 platform. 
> Currently Rancher only supports Kubernetes version `v1.12` and above  on ARM64 platform.

File: content/rancher/v2.x/en/arm64-platform/_index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+For the ARM64 platform, Rancher currently supports the following two features:  
+
+- Running Rancher Server on ARM64 platform
+- Using Rancher to manage ARM64 Platform Kubernetes
+    - Import ARM64 Kubernetes clusters
+    - Create ARM64 Kubernetes clusters with custom nodes
+
+>**Notes:**
+>The `Monitoring`,`Alerts and Notifiers`,`Pipelines`,`Logging` features are currently not supported in the ARM64 platform.
```
Comment (loganhz) [inline content/rancher/v2.x/en/arm64-platform/_index.md]: Add this here to make it obvious

> Currently Rancher only supports using `Flannel` network plugin to create Kubernetes clusters on ARM64 platform. 
> Currently Rancher only supports Kubernetes version `v1.12` and above  on ARM64 platform.

File: content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md (hunk 5 above)
Comment (jianghang8421) [reply content/rancher/v2.x/en/arm64-platform/manage-arm64-k8s/_index.md]: Updated

File: content/rancher/v2.x/en/arm64-platform/_index.md (hunk 6 above)
Comment (jianghang8421) [reply content/rancher/v2.x/en/arm64-platform/_index.md]: Updated

**Comment by loganhz** [comment]
Please add `_Available as of v2.2.0_` so that people know when these features were available.

https://github.com/rancher/docs/pull/1199/files#diff-8ad9c71b76b83c64e18dda58fa1e8c68R9

**Comment by loganhz** [comment]
LGTM

File: content/rancher/v2.x/en/arm64-platform/_index.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+---
+title: ARM64 Platform Support (Experimental)
+weight: 7600
+---
+
+_Available as of v2.2.0_
+
+From Rancher v2.2.0, you can run Rancher server on ARM64 platform and manage ARM64 Kubernetes clusters. There is no extra configuration needed for ARM64. You can do the same on ARM64 platform as AMD64.
```
Comment (ankur111) [inline content/rancher/v2.x/en/arm64-platform/_index.md]: are you trying to say that ARM64 and AMD64 config requirements are the same?
If yes, please change "You can do the same on ARM64 platform as AMD64." to "The same instructions also apply to AMD64"

Comment (loganhz) [reply content/rancher/v2.x/en/arm64-platform/_index.md]: I am trying to say `The instructions for ARM64 are the same with AMD64.`

Comment (loganhz) [reply content/rancher/v2.x/en/arm64-platform/_index.md]: So I changed to `The instructions are the same with AMD64.` is that correct?

Comment (ankur111) [reply content/rancher/v2.x/en/arm64-platform/_index.md]: The instructions are same for AMD64

Comment (loganhz) [reply content/rancher/v2.x/en/arm64-platform/_index.md]: updated

**Comment by erikwilson** [comment]
Catalog support may not work well arm64, most apps will likely fail to install without some more work in that area - should probably add a note somewhere with that information.

**Pull request merged**